.github/workflows/build_release.yml

```diff
@@ -0,0 +1,72 @@
+name: Build Release
+description: Build release bundle and upload to GitHub release
+
+on:
+  release:
+    types: [published]
+  workflow_dispatch:
+
+jobs:
+  build-release:
+    name: Build Release
+    runs-on: macos-26
```

Comment:
Running on GitHub runners instead of self hosted, because the self-hosted runners are not available for public repositories.